pg_cron.control

```diff
@@ -2,4 +2,3 @@ comment = 'Job scheduler for PostgreSQL'
 default_version = '1.6'
 module_pathname = '$libdir/pg_cron'
 relocatable = false
-schema = pg_catalog
```

Comment:
To be honest, I don't think this is an acceptable change. We've had privilege escalation issues in the past due to public schema. Allowing any schema is even worse.

Reply:
Maybe if we do a very careful audit:

e.g. the unqualified nextval default in
https://github.com/citusdata/pg_cron/blob/main/pg_cron--1.2--1.3.sql looks dodgy

(It's easy to miss things)

Reply:
After reading through the issue #113, I no longer think that we should update our control file.

I opened a separate PR that qualifies the nextval call in the migration script in #291 and I plan to close this one without merging.